templates/blog.html

```diff
@@ -22,9 +23,7 @@
 {% endblock %}
 {% block gutter %}
 <div class="col-sm-12 col-md-4 gutter">
-  <h3><p><a href="/news/buzz/atom.xml"><i class="fa fa-rss-square" aria-hidden="true"></i> RSS Feed</a></p></h3>
```

Comment:
No, it lives in the gutter top now
https://github.com/pybee/pybee.github.io/pull/164/files/b8a67f7c262350953a242d927515124f808c3af2#diff-827e86c354c59f395efb5c3f1f2e6ee5R26